libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/internal/camera/data/MapboxNavigationViewportDataSourceDebugger.kt

```diff
@@ -0,0 +1,256 @@
+package com.mapbox.navigation.ui.maps.internal.camera.data
```

Comment:
Should `MapboxNavigationViewportDataSourceDebugger` be moved out the `internal` package?